tests/helpers.py

```diff
@@ -982,3 +995,99 @@ def delete_deploymentconfig(pod_obj):
     """
     dc_ocp_obj = ocp.OCP(kind=constants.DEPLOYMENTCONFIG)
     dc_ocp_obj.delete(resource_name=pod_obj.get_labels().get('name'))
+
+
+def delete_objs_parallel(obj_list):
+    """
+    Deletes objs specified in list
+    Args:
+        obj_list(list): List can be obj of pod, pvc, etc
+
+    Returns:
+        bool: True if obj deleted else False
+
+    """
+    with ThreadPoolExecutor(32) as p:
+        for obj in obj_list:
+            p.submit(obj.delete)
+    return True
+
+
+def memory_leak_analysis(median_dict):
+    """
+    Memory leak analysis after the execution of test case
+    Memory leak is analyzed based on "RES" value, i.e. list[7] in code
```

Comment:
I can see other usages of this function but based on the description it will be difficult to know how to use it.
Please try to elaborate more about the median_dict and even add some example of such dict

Reply:
@RazTamir  added more comments for usage, would request to review.